### ISSUE TITLE
Add Support For A Queue Role

### ISSUE THREAD

**Issue by aokennard** [pattyjogal/cypher-bot#14]
To avoid unnecessary @'ing for all 10mans players (or manually going through the queue and @'ing people), there should be support for a dynamic queue role for those added in a queue. @'ing the role only @'s people in the queue. Adding up assigns the role, and it can be removed on queue empty. This can also be extended to only @ people not in voice channel(s).

**Comment by JoshSanch** [comment]
Good call - will require support for admins closing the queue via the bot. Making an issue for that now that will need to be addressed before this one.

**Comment by pattyjogal** [comment]
I think we could either do this or just have cypher @ the people in the specific queue, since we are planning to have multiple queues at some point

**Comment by JoshSanch** [comment]
That's true, although I think having the bot manage some queue roles would be nicer from a user perspective. Seeing all of those ats is going to be ugly and it'll be very obvious to the users what queue they're getting pinged for if they can literally see something along the lines of `@Queue1`.

**Comment by pattyjogal** [comment]
Actually there is an added benefit of creating multiple queue roles in that you can use them in discord for the duration of the queue. It is going to be harder to write to ensure they are cleaned up smoothly, bear in mind.
